src/NzbDrone.Core/Localization/Core/en.json

```
@@ -534,6 +534,12 @@
   "DownloadClientStatusSingleClientHealthCheckMessage": "Download clients unavailable due to failures: {downloadClientNames}",
   "DownloadClientTransmissionSettingsDirectoryHelpText": "Optional location to put downloads in, leave blank to use the default Transmission location",
   "DownloadClientTransmissionSettingsUrlBaseHelpText": "Adds a prefix to the {clientName} rpc url, eg {url}, defaults to '{defaultUrl}'",
+  "DownloadClientTriblerSettingsAnonymityLevel": "Anonymity level",
+  "DownloadClientTriblerSettingsAnonymityLevelHelpText": "Number of proxies to use when downloading content. To disable set to 0. Proxies reduce download/upload speed. See https://www.tribler.org/anonymity.html",
+  "DownloadClientTriblerSettingsApiKeyHelpText": "Api key, found in %APPDATA%\\Roaming\\.Tribler\\7.14\\triblerd.conf, the api key is [api].key, NOT [http_api].key",
```

```suggestion
  "DownloadClientTriblerSettingsApiKeyHelpText": "[api].key from triblerd.conf",
```

tribler is multi-platform so it doesn't make sense to call out a windows path.